single_tech_samples/synapseanalytics/sample3_integration_testing/tests/integrationtests/dataconnectors/adls.py

```diff
@@ -0,0 +1,231 @@
+        print(e)
+        raise
+
+
+def read_parquet_file_from_ADLS(
+    adls_connection_client, container: str, file_name: str, base_path: str = None
+):
+    """Download file from ADLS and convert to parquet data frame
+
+    Args:
+        adls_connection_client (DataLakeServiceClient): ADLS Connection Object
+        container (str): Container Name where file needs to be uploaded
+        base_path (str): Base Folder Path in ADLS where file will be uploaded
+        file_name (str): File Name to Upload
+
+    Returns:
+        processed_df (bytes): DataFrame Of Read File
+    """
+    downloaded_bytes = download_from_ADLS(
+        adls_connection_client, container, file_name, base_path
+    )
+    processed_df = get_parquet_df_from_contents(downloaded_bytes)
+    return processed_df
+
+
+def cleanup_ADLS_files(
+    adls_connection_client: DataLakeServiceClient,
+    containers: List[str],
+    cleanup_folders: List[str],
+):
+    """Delete containers in ADLS
+
+    Args:
+        adls_connection_client (DataLakeServiceClient): ADLS Connection Object
+        containers (List[str]): Container Name
+        cleanup_folders (List[str]): Folders to be Deleted  in ADLS
+    """
+
+    for container in containers:
+        for folder in cleanup_folders:
+            _delete_folder(adls_connection_client, container, folder)
+
+
+def _delete_folder(
+    datalake_service_client: DataLakeServiceClient, container: str, folder: str
+):
+    """Delete a folder in ADLS
+
+    Args:
+        datalake_service_client (DataLakeServiceClient): ADLS connection client
+        container (str): Container Name
+        folder (str): Folder to be deleted
+    """
+    try:
+        print(f"DELETING FOLDER {folder} from ADLS")
+        file_system_client = datalake_service_client.get_file_system_client(
+            file_system=container
+        )
+        directory_client = file_system_client.get_directory_client(folder)
+        directory_client.delete_directory()
+
+    except Exception as e:
+        print(f"Failed on deleting folder {container}/{folder} with {e}")
+
+
+def _delete_file(
+    datalake_service_client: DataLakeServiceClient,
+    container: str,
+    base_path: str,
+    file_name: str,
+):
+    """Delete a file in ADLS
+
+    Args:
+        datalake_service_client (DataLakeServiceClient): ADLS connection client
+        container (str): Container Name
+        base_path (str): Path to file under the container to be deleted
+        file_name (str): File to be deleted
+    """
+    try:
+        print(f"DELETING File {file_name} from ADLS")
+        file_path = f"{container}/{base_path}/{file_name}"
+        file_system_client = datalake_service_client.get_file_system_client(
+            file_system=container
+        )
+        file_client = file_system_client.get_file_client(file_path)
+        file_client.delete_file()
+
+    except Exception as e:
+        print(f"Failed on deleting folder {file_path} with {e}")
```

Comment:
incorrect echo, should likely echo 'failed on deleting file' rather than 'folder'